modules/vye/app/controllers/vye/application_controller.rb

```diff
@@ -2,8 +2,6 @@
 
 module Vye
   class ApplicationController < ::ApplicationController
-    include Pundit::Authorization
```

Comment:
@binq Just want to make sure you see this, I dunno what you decided about Pundit in the application controller but could have ramifications if it's removed

Reply:
Yeah I think this should be ok since it's a redundant include. I included it by mistake since it's already in at app/controllers/application_controller.rb:12.